src/legendoptics/tyvek.py

```diff
@@ -0,0 +1,45 @@
+"""
+Tyvek reflector.
+"""
+
+from __future__ import annotations
+
+import logging
+
+import pint
+from pint import Quantity
+
+from legendoptics import store
+from legendoptics.utils import readdatafile
+
+log = logging.getLogger(__name__)
+u = pint.get_application_registry()
+
+
+@store.register_pluggable
+def tyvek_reflectivity() -> tuple[Quantity, Quantity]:
+    """Tyvek reflectivity from [Janacek2012]_.
+
+       A little bit more conservative than in the paper
+       (-1% reflectivity compared to the values in the paper).
+
+    .. optics-plot::
+    """
+    return readdatafile("tyvek_reflectivity.dat")
+
+
+def pyg4_tyvek_attach_border_params(mat, reg, reflectivity_scale: float = 1) -> None:
```

Comment:
```suggestion
def pyg4_tyvek_attach_reflectivity(mat, reg, reflectivity_scale: float = 1) -> None:
```
I would still rename this to make it more in line with the rest of the library (also rename it in the tests)

Reply:
done